packages/strapi-plugin-content-manager/admin/src/containers/EditPage/saga.js

```diff
@@ -155,7 +155,7 @@ export function* submit() {
     yield put(submitSuccess());
 
   } catch(err) {
-    if (isArray(err.response.payload.message)) {
+    if (isArray(get(err, 'responsepayload.message'))) {
```

Comment:
Thanks @Aurelsicoko!

```suggestion
    if (isArray(get(err, 'response.payload.message'))) {
```